### DISCUSSION TITLE
Authentication with RabbitMQ failed

### DISCUSSION THREAD

**Discussion by vulnfind** [postalserver/postal#2417] [Help with using Postal]
both the worker and requeuer process is not starting because the authentication with rabbit mq is failing and all my emails are in pending status

![image](https://user-images.githubusercontent.com/100760294/233335197-364fdbf5-331a-4df5-a01c-aef9be795df4.png)

![image](https://user-images.githubusercontent.com/100760294/233335417-af47a667-4d17-495e-a673-8aa430a1ce2d.png)

**Comment by willpower232** [comment]
If login was refused, does that mean you didn't enter the right information into the config file? If you did, did you need to restart Postal to apply the latest version of the config file?
